src/components/constants/Pokemon.ts

```diff
@@ -200,8 +291,15 @@ export class Pokemon {
             "info": [
                 "Using its ability to read minds, it will identify impending danger and TELEPORT to safety."
             ],
-            "evolvesInto": "Kadabra",
-            "evolvesVia": "Level 16",
+            "evolutions": [{
+			    "evolvedForm": "Kadabra",
```

Comment:
I don't know if you saw in the last review - it's probably better if you don't include all the new evolutions in this PR. Instead we should solidify on the format.

But, since the format LGTM, if you want to add them here instead of a followup PR that's fine too.